### PR TITLE
release snowflake notebook updates

### PR DESCRIPTION
**Pull request by jameslamb**
This PR proposes merging the changes on `main` into the current release, so they can get out to customers more quickly.

### How I tested this

I clone this branch of this repo into `app.daily`, and ran the notebooks changed in this PR manually.  Since the daily integration tests are already running against `main`, I thought "oh ok, I only need to test the differences between this PR and main".

### CODE REVIEW

File: examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb
Comment (jameslamb) [inline examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb:375]: this isn't necessary (no customers on current releases should be facing this), but it avoids this bug: https://github.com/saturncloud/dask-saturn/pull/54

I think it's worth adding here to be sure people who get these notebooks but `dask-saturn` 0.2.2 don't hit the error described in that PR.

**Comment by jameslamb** [comment]
Ok I think this is ready for review!

**Comment by jameslamb** [comment]
I think we should wait on this until #91 is merged to `main`, and then merge that into this PR. That fixes the last remaining known issue in the example notebooks.

File: examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb
Comment (jameslamb) [reply examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb:375]: I did test this live on `app.daily` (should have mentioned that before) and confirmed that it avoided the `dask-saturn` bug mentioned above

**Comment by jameslamb** [comment]
Ok I think this is ready for review now that #91 is merged into it.

The only change I made in this PR is https://github.com/saturncloud/examples/pull/90#discussion_r574082177. Otherwise, the rest of this diff is all just from previously-merged PRs on `main` that aren't on release yet.

**Review by jsignell — APPROVED** [review]
Thanks for doing this!

**Pull request merged**
